components/director/internal/tenantfetcher/event_page_test.go

```diff
@@ -332,7 +333,7 @@ func Test_getTenantMappings(t *testing.T) {
 		t.Run(test.name, func(t *testing.T) {
 			events := make([][]byte, 0, len(test.detailsPairs))
 			for i, detailPair := range test.detailsPairs {
-				events = append(events, fixEventWithDetails(fmt.Sprintf("id%d", i), fmt.Sprintf("foo%d", i), "GlobalAccount", constructJSONObject(detailPair...), test.fieldMapping))
+				events = append(events, fixEventWithDetailsWithoutGA(fmt.Sprintf("id%d", i), fmt.Sprintf("foo%d", i), "GlobalAccount", constructJSONObject(detailPair...), test.fieldMapping))
```

Comment:
The differentiation between GA events and SA events can be removed here as well.

Reply:
Yes, I just adapted it after discussing the other occurrence in the service tests.